sp/src/game/server/mapbase/ai_monitor.cpp

```diff
@@ -672,6 +672,30 @@ int CAI_Monitor::TranslateScheduleString(const char *schedName)
 	return 0;
 }
 
+template<typename Translator>
+static void SetForEachDelimited( CAI_Monitor &monitor, const char *szValue, const char *delimiters, void (CAI_Monitor::*setter)(int), Translator translator)
```

Comment:
This and `CAI_MonitorTranslator` look kinda awkward, to deal with the fact that `translator` may either be a member-function (`TranslateConditionString`, `TranslateScheduleString`) or a free-function (`atoi()`).
All because I wanted to avoid copy-pasting this code two additional times... I think that's fair enough, though YMMV, in which case I can remove the templatery and just copy-paste this for the three invocations below.